Unbreak with libxml2 >=2.15.0
../gupnp-tools-0.12.2/src/common/pretty-print.c:47:21: error: '__xmlIndentTreeOutput' is deprecated [-Werror,-Wdeprecated-declarations]

Index: meson.build
--- meson.build.orig
+++ meson.build
@@ -44,7 +44,6 @@ conf.set('HAVE_GTK_SOURCEVIEW', gtksourceview.found())
 conf.set('GETTEXT_PACKAGE', '"@0@"'.format(meson.project_name()))
 conf.set('LOCALEDIR', '"@0@"'.format(localedir))
 cc = meson.get_compiler('c')
-add_project_arguments(cc.get_supported_arguments('-Werror=deprecated-declarations'), language: 'c')
 conf.set('GLIB_VERSION_MIN_REQUIRED', 'GLIB_VERSION_@0@'.format(glib_version.underscorify()))
 conf.set('GLIB_VERSION_MAX_ALLOWED', 'GLIB_VERSION_@0@'.format(glib_version.underscorify()))
 
